botler_api.js

```diff
@@ -0,0 +1,42 @@
+
+var request = require('request');
+
+var botlerUrl = "http://localhost:3000/";
+
+module.exports = {
+  createExpenses: createExpenses
+};
+
+function createExpenses(expenses, auth, logger, handleError) {
```

Comment:
@ronaldofs Is there a way to pass these dependencies (`logger`, `handleError`, etc) in a *cleaner* way?